x-pack/test/functional/services/ml/job_table.ts

```diff
@@ -50,26 +50,43 @@ export function MachineLearningJobTableProvider(
   const retry = getService('retry');
 
   return new (class MlJobTable {
-    public async clickJobRowCalendar(jobId: string, calendarId: string) {
-      await this.withDetailsOpen(jobId, async function clickJobRowCalendar() {
-        const calendarSelector = `${jobId}-${calendarId}`;
-        await testSubjects.existOrFail(calendarSelector, {
-          timeout: 3_000,
-        });
-        await testSubjects.click(calendarSelector, 3_000);
-      });
+    public async assertCalendarSettingsPage(jobId: string, calendarId: string): Promise<void> {
```

Comment:
IMO this method name can be a bit confusing, it's not clear what action is taken. I'd suggest to name it closer to its original name, e.g. something like `clickJobRowCalendarWithAssertion`.

Reply:
I hope [this](https://github.com/elastic/kibana/pull/178553/commits/9b11596c5876671dc8a9732cc5e58b2ced011a99) satisfies boss!